RFC/readme.md

```diff
@@ -0,0 +1,142 @@
+# Welcome
+
+### About this Repository
```

Comment:
should this be ## instead of ###

Reply:
This was a visual choice rather than rules of how a markdown has to follow.

Reply:
I think the visual should be looked at after rendering and not the raw file

Reply:
Yeah, I did the choice based on the rendered file. Not the raw.